File: src/test/scala/ch/netzwerg/scala/PersonTest.scala
package ch.netzwerg.scala

import org.scalatest._

class PersonTest extends FlatSpec with Matchers {

  val Name = "Ada"

  "A Person" should "be constructable" in {
    val person = Person(Name)
    person.name should equal(Name)
    person should equal(Person(Name))
  }

}

File: src/test/java/ch/netzwerg/java/PersonTest.java
package ch.netzwerg.java;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class PersonTest {

    private static final String NAME = "Ada";

    @Test
    public void construction() {
        Person person = new Person(NAME);
        assertEquals(NAME, person.getName());
        assertEquals(person, new Person(NAME));
    }

}